(5.0) Clustering with DDEV: supported Elasticsearch versions (#3240)
ezsystems/developer-documentation: docs/infrastructure_and_maintenance/clustering/clustering_with_ddev.md

The following sequence of commands:

1. Adds the Elasticsearch container

> 2. Sets the Elasticsearch version to 8 (default is 9 which is not supported, 7 is supported)

Author: ```suggestion
2. Sets the Elasticsearch version to supported 8 - a full version number is required, see [Elasticsearch Docker image](https://hub.docker.com/_/elasticsearch)
```

> 2. Sets the Elasticsearch version to supported 8 - a full version number is required, see [Elasticsearch Docker image](https://hub.docker.com/_/elasticsearch)

Author: ```suggestion
2. Sets the Elasticsearch version to 8 (default is 9 which is not supported, 7 is supported) - a full version number is required, see [Elasticsearch Docker image](https://hub.docker.com/_/elasticsearch)
```

3. Sets Elasticsearch as the search engine
4. Restarts the DDEV cluster and clears application cache
5. Injects the schema and reindexes the content

```bash
ddev add-on get ddev/ddev-elasticsearch

> ddev dotenv set .ddev/.env.elasticsearch --elasticsearch-docker-image=elasticsearch:8.19.0

Author: ```suggestion
ddev dotenv set .ddev/.env.elasticsearch --elasticsearch-docker-image=elasticsearch:8.19.18
```
